extensions/amp-form/0.1/amp-form.js

```diff
@@ -557,6 +563,7 @@ export class AmpForm {
         const shouldSubmitFormElement = !event;
 
         this.handleNonXhrGet_(shouldSubmitFormElement);
+        this.dirtinessHandler_.onSubmitSuccess();
```

Comment:
Why we need to hook here?

Reply:
For non-XHR GET forms, especially the ones that open in a new window, i.e. `target=_blank`). Otherwise the form is stuck in `submitting` state, and will never appear dirty.

Reply:
Yeah I am not very familiar with this code path, I'll just let Carlos check this.

Reply:
This looks right to me 👍